src/mmw/js/src/core/models.js

```diff
@@ -101,6 +101,14 @@ var MapModel = Backbone.Model.extend({
         this.set('size', updatedSize);
     },
 
+    toggleSecondarySidebar: function() {
+        var sizeCopy = _.clone(this.get('size')),
```

Comment:
Do we need to use `_.clone` here? Could we not have simply done `var size = this.get('size')`?

Reply:
Same as`toggleSidebar` above, [here's our discussion](https://github.com/WikiWatershed/model-my-watershed/pull/1890#discussion_r116815092) on how we need to trigger a change event, and the `merge` oddly doesn't suffice. I'll test this out again to confirm...

Reply:
Ah I see. Very well, let's keep it. Good to know I'm hard headed as ever.